lib/her/model/attributes.rb

```diff
@@ -31,15 +31,25 @@ def initialize(attributes={})
 
         attributes = self.class.default_scope.apply_to(attributes)
         assign_attributes(attributes)
+        _clear_changes_information unless new?
         yield self if block_given?
         run_callbacks :initialize
       end
 
+      def _clear_changes_information
```

Comment:
We've actually been meaning to update activemodel for some time. 4.2.1 or 4.2.7 will give us the same [api](https://apidock.com/rails/ActiveModel/Dirty) and help clean up some of the existing code.